modules/apps/oauth2-provider/oauth2-provider-test/src/testIntegration/java/com/liferay/oauth2/provider/client/test/TokenIntrospectionTest.java

```diff
@@ -74,9 +74,10 @@ public void testTokenIntrospectionApplicationCode() {
 		Response response = invocationBuilder.post(Entity.form(formData));
 
 		Assert.assertEquals(
-			applicationClientId, parseJsonField(response, "client_id"));
+			Response.Status.OK.getStatusCode(), response.getStatus());
```

Comment:
@stian-sigvartsen I think so considering that we were not testing anything at all about the Token Introspection. 
With the new CXF you can't get from a response an entity more than once in the thread, I think it's fine with checking that it returns a 200 Status and one of the expected values instead of [having to refactor this](https://github.com/liferay-appsec/liferay-portal/blob/master/modules/apps/oauth2-provider/oauth2-provider-test/src/testIntegration/java/com/liferay/oauth2/provider/client/test/BaseClientTestCase.java#L630) common util class to extract several fields.

Reply:
That's a sensible argument I think. Thanks!